### PR TITLE
feat: Add to the API the last_update for tasks

### PR DESCRIPTION
**Pull request by kairoaraujo**
It adds to the API responses for HTTP CODE 202 (received), used by tasks, the field `last_update`.

This field helps to the requesters manage the tasks based on date time

Signed-off-by: Kairo de Araujo <kdearaujo@vmware.com>

### CODE REVIEW

**Review by MVrachev — COMMENTED** [review]
Can you describe how would you use this new feature knowing when there was the last update?
Can you describe the idea a little more?

**Review by rdimitrov — APPROVED** [review]

**Comment by rdimitrov** [comment]
> Can you describe how would you use this new feature knowing when there was the last update?
> Can you describe the idea a little more?

Details from our discussion with Kairo - 
> Use it to measure the time from sending the add target request to being able to query for that target successfully in X amount of time. Also, testing things like sending 1000 new targets in 10 requests and measuring their processing
> To sum it up - it can be used for things like performance testing, observability, etc.

File: repository_service_tuf_api/targets.py [hunk 1]
```diff
@@ -18,6 +19,7 @@
 class ResponseData(BaseModel):
     targets: List[str]
     task_id: str
+    last_update: datetime
```
[the remark this answers is not in the document]
Comment (kairoaraujo) [reply repository_service_tuf_api/targets.py:22]: I think we can keep `last_update`.
You can find many uses of `last_update` in the APIs.
Some examples

- https://developer.salesforce.com/docs/atlas.en-us.216.0.api_analytics.meta/api_analytics/sforce_analytics_rest_api_get_fields.htm

- https://gudhub.com/docs/rest-api/items/

- https://www.ibm.com/docs/en/psww2500/2.3.3?topic=management-virtual-application-instances

- https://publichealthengland.github.io/coronavirus-dashboard-api-python-sdk/pages/examples/timestamps.html

[the remark this answers is not in the document]
Comment (MVrachev) [reply repository_service_tuf_api/targets.py:22]: I can see the reasons behind `last_update`.
Okay, let's leave it like that.

**Review by MVrachev — APPROVED** [review]

**Pull request merged**